File: c/3-loops/workout/lw-3.c
#include <stdio.h>
int main()
{
    int a,b;
    int sum =0;
    scanf("%d %d\n",&a,&b);
    if(a<0||b<0||a>b)
    {
        printf("Invalid input\n");
    }
    else
        {
            for(int i =a;i<=b;i++)
            {
                if(i%7==0)
                {
                    sum+=i;
                }
            }printf("%d\n",sum);
        }
}

/

File: python/7-lists/workout/lw-3.py
try:
    a = list(map(int, input().split()))
    unique_scores = list(set(a))

    if len(unique_scores) < 2:
        print("Invalid input")
    else:
        unique_scores.sort(reverse=True)
        print(unique_scores[1])
except:
    print("Invalid input")